tests/investtab.test.js

```diff
@@ -192,29 +187,34 @@ describe('Save InVEST Model Setup Buttons', () => {
 
 describe('InVEST Run Button', () => {
   const spec = {
-    module: 'natcap.invest.foo',
-    model_name: 'Foo Model',
+    module: 'natcap.invest.bar',
+    model_name: 'Bar Model',
     args: {
       a: {
-        name: 'afoo',
+        name: 'abar',
         type: 'freestyle_string',
       },
       b: {
-        name: 'bfoo',
+        name: 'bbar',
         type: 'number',
       },
       c: {
-        name: 'cfoo',
+        name: 'cbar',
         type: 'csv',
       },
     },
   };
 
   beforeAll(() => {
     getSpec.mockResolvedValue(spec);
+    // mock out the whole UI config module
+    jest.mock('../src/ui_config', () => {
+      return {'Bar Model': {order: [['a'], ['b', 'c']]}}

```

Comment:
Same comment, maybe these strings could be references to the `spec` to avoid duplication